_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-videopress.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * REST API endpoint for managing VideoPress metadata.
+ *
+ * @package Jetpack
+ * @since 9.2.0
+ */
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * VideoPress wpcom api v2 endpoint
+ */
+class WPCOM_REST_API_V2_Endpoint_VideoPress extends WP_REST_Controller {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'videopress';
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register the route.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/meta',
+			array(
+				'args'                => array(
+					'guid'          => array(
+						'description' => __( 'The VideoPress video guid.', 'jetpack' ),
+						'type'        => 'string',
+						'required'    => true,
+					),
+					'title'         => array(
+						'description' => __( 'The title of the video.', 'jetpack' ),
+						'type'        => 'string',
+						'required'    => false,
+					),
+					'description'   => array(
+						'description' => __( 'The description of the video.', 'jetpack' ),
+						'type'        => 'string',
+						'required'    => false,
+					),
+					'rating'        => array(
+						'description' => __( 'The video content rating. One of G, PG-13, R-17 or X-18', 'jetpack' ),
+						'type'        => 'string',
+						'required'    => false,
+					),
+					'display_embed' => array(
+						'description' => __( 'Display the share menu in the player.', 'jetpack' ),
+						'type'        => 'boolean',
+						'required'    => false,
+					),
```

Comment:
Maybe we can add `sanitize_callback` to all those arguments, to clean things up a bit before it gets sent to WordPress.com?